src/core/BufferGeometry.js

```diff
@@ -952,7 +952,30 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 			var attribute = attributes[ key ];
 
-			var array = Array.prototype.slice.call( attribute.array );
+			var array;
+
+			if ( attribute.isInterleavedBufferAttribute ) {
```

Comment:
This is necessary since some `glTF` models might have interleaved buffer data like in `webgl_animation_skinning_blending`. This code section creates normal `BufferAttribute`s in order to ensure correct data.